lib/Zend/Locale/Data.php

```diff
@@ -946,11 +960,13 @@ public static function getList($locale, $path, $value = false)
         }
 
         if (isset(self::$_cache)) {
+            $data = serialize($temp);
             if (self::$_cacheTags) {
-                self::$_cache->save( serialize($temp), $id, array('Zend_Locale'));
+                self::$_cache->save( $data, $id, array('Zend_Locale'));
             } else {
-                self::$_cache->save( serialize($temp), $id);
+                self::$_cache->save( $data, $id);
             }
+            static::$_localCache['id'] = $data;
```

Comment:
@tmotyl 
why this key used `'id'` ?
should it be the `$id` variable ?

Reply:
damn, it should. Can you make a PR?

Reply:
and in line 1526 too

Reply:
one reason, why its always good to wait for a second Review 👀